test/long-stack-traces.js

```diff
@@ -0,0 +1,138 @@
+import test from 'ava';
+import StackUtils from '../';
+import longStackTraces from './fixtures/long-stack-traces';
+import pify from 'pify';
+const nestedErrors = pify(require('./fixtures/nested-errors'), Promise);
+
+import {join, fixtureDir} from './_utils';
+
+function internals() {
+	return StackUtils.nodeInternals().concat([
+		/[\\\/]long-stack-traces\.js:[0-9]+:[0-9]+\)?$/,
+		/[\\\/]internal-error\.js:[0-9]+:[0-9]+\)?$/,
+		/[\\\/]internal-then\.js:[0-9]+:[0-9]+\)?$/,
+		/[\\\/]node_modules[\\\/]/,
+		// TODO: Should any of these be default internals?
+		/[\\\/]\.node-spawn-wrap-\w+-\w+[\\\/]node:[0-9]+:[0-9]+\)?$/,
+		/internal[\\\/]module\.js:[0-9]+:[0-9]+\)?$/,
+		/node\.js:[0-9]+:[0-9]+\)?$/
```

Comment:
I think the above three lines should be part of the default internals.

The `spawn-wrap` is a bit iffy, but I doubt anyone actually wants that